src/deluge/util/fixedpoint.h

```diff
@@ -24,6 +25,7 @@ typedef int32_t q31_t;
 #define ONE_Q15 65536
 #define NEGATIVE_ONE_Q31 -2147483648
 #define ONE_OVER_SQRT2_Q31 1518500250
+#if !defined(i386)
```

Comment:
at some point it would be good to break these out to separate header probably, i can see us wanting to support e.g. aarch64 and x86_64 as well. that can be followup work though.

Reply:
Makes sense - changing to check for 32 bit arm instead for future flexibility